src/app/components/Item/ItemHoldings.jsx

```diff
@@ -66,7 +66,7 @@ class ItemHoldings extends React.Component {
     e.preventDefault();
     this.props.updateIsLoadingState(true);
 
-    trackDiscovery('Bib Item Request', `Item Details - ${itemId}`);
+    trackDiscovery('Item Request', `Item Details`);
```

Comment:
You don't need to use back ticks anymore since you're not using a variable in the string.